api/kronos.js

```diff
@@ -45,6 +45,22 @@ export default class {
     return data;
   }
 
+  // ====================
+  // Create Contact
+  // ====================
+  async createContact(contact) {
+    const data = await this.http.post(`api/v2018/organizations/${encodeURIComponent(contact.organizationId)}/contacts`, { json: contact }).json();
```

Comment:
please passe the `organizationId` as 1st parameter,  `contact.organizationId ` is not a field of contact update
```suggestion
  async createContact(organizationId, contact) {
    const data = await this.http.post(`api/v2018/organizations/${encodeURIComponent(organizationId)}/contacts`, { json: contact }).json();
```